CHANGELOG.md

```diff
@@ -8,24 +8,24 @@ This file is used to list changes made in each version of the AWS ParallelCluste
 
 **ENHANCEMENTS**
 - Enable clusters to authenticate users by integrating with Active Directory (AD) domains managed via AWS Directory Service.
-- Install NVIDIA and CUDA for ARM.
+- Install NVIDIA driver and CUDA library for ARM.
```

Comment:
Isn't it a set of drivers, so NVIDIA drivers

Reply:
done